.github/workflows/test_cli_action.yaml

```diff
@@ -3,15 +3,23 @@ name: "CLI Action testing"
 on:
   schedule:
     - cron: "0 0 * * *"
+  workflow_dispatch:
```

Comment:
Empty workflow_dispatch?

Reply:
Takes no input, allows triggering the workflow manually